05/DB.js

```diff
@@ -5,34 +5,32 @@ export default class DB {
 
     insert(data) {
         return new Promise((resolve, reject) => {
-            if(data.id) {
-                if(typeof data.id !== 'number') {
-                    this.async(reject,'ID can be only number!');
-                    return null; // stop function
```

Comment:
Nie powinieneś tego usuwać - w ten sposób testy nie działają jak trzeba.
Chodzi o to, że jak pojawi się błąd to nie chcemy wykonywać dodatkowych operacji.
Jeśli nie ma tego `return` to funkcja działa dalej.  W zależności od szybkości wykonywania operacji może się zdarzyć, że szybciej wykona się `resolve(data)` i wtedy zamiast błędu mamy dane (czego nie powinno być).